mon-pix/app/controllers/assessments/challenge.js

```diff
@@ -18,7 +18,7 @@ export default class ChallengeController extends Controller {
   @tracked competenceLeveled = null;
   @tracked challengeTitle = defaultPageTitle;
   @tracked hasFocusedOutOfChallenge = false;
-  @tracked hasFocusedOutOfWindow = false;
+  @tracked hasFocusedOutOfWindow = false || this.model.assessment.hasFocusedOutChallenge;
```

Comment:
Équivalent à 
```suggestion
  @tracked hasFocusedOutOfWindow = this.model.assessment.hasFocusedOutChallenge;
```
?
Ou bien faut il gérer l'`undefined`: 
```suggestion
  @tracked hasFocusedOutOfWindow = this.model.assessment.hasFocusedOutChallenge || false;
```

Reply:
Si c'est pour s'assurer qu'on a pas du null ou undefined le plus juste ce serait ça :

```suggestion
  @tracked hasFocusedOutOfWindow = this.model.assessment.hasFocusedOutChallenge ?? false;
```
Mais est-ce qu'on l'a le `??`

Reply:
`this.model.assesment.hasFocusOutChallenge` ne semble jamais être `null` ou `undefined`, on reste sur la 1ère solution mais on garde les autres propositions en tête si jamais.